source/upgrade.txt

```diff
@@ -0,0 +1,95 @@
+.. _cpp-upgrade:
+
+=======================
+Upgrade Driver Versions
+=======================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: compatibility, backwards compatibility
+
+Overview
+--------
+
+This page describes the changes you must make to your application
+when you upgrade to a new version of the {+driver-short+}.
+
+Before you upgrade, perform the following actions:
+
+- Ensure the new {+driver-short+} version is compatible with the {+mdb-server+} versions
+  your application connects to and the C++ standard version your
+  application compiles with. For version compatibility information, see the
+  :ref:`{+driver-short+} Compatibility <cpp-compatibility-tables>`
+  page.
+- Address any breaking changes between the driver version
+  your application is using and your planned upgrade version in the
+  :ref:`Breaking Changes <cpp-breaking-changes>` section.
+
+.. tip::
+
+   To ensure compatibility across {+mdb-server+} versions when
+   upgrading driver versions, use the
+   :ref:`{+stable-api+} <cpp-stable-api>`.
+
+.. _cpp-breaking-changes:
+
+Breaking Changes
+----------------
+
+.. note::
+
+   The {+driver-short+} follows :wikipedia:`semantic versioning. <Software_versioning#Schemes>`
+   The driver might break build system or package compatibility in any release, and
+   patch releases can revert accidental API breaking changes.
```

Comment:
```suggestion
   The driver may break build system or package compatibility in any release.
   Patch releases may revert accidental API breaking changes.
```
Recommend using "may" instead of "might/can" to imply deliberate policy rather than possibility. Also recommend splitting into individual sentences to better separate the two distinct cases.

Reply:
Confirmed usage of "may" with my team and this should be fine